### ISSUE TITLE
Text string with emoji crashes the karma

### ISSUE THREAD

**Issue by fetis** [karma-runner/karma-junit-reporter#171]
This bug is triggered by the next Jasmine test

```javascript
it('should accept emoji', () => {
  expect('😀').toBe('1');
});
```
### Expected behaviour
Print fail output and contine running

### Actual behaviour
Prints fail output and crashes with error
`ERROR [karma-server]: Error: Invalid character (�) in string: Expected '😀' to be '1'.`

### Environment Details

Karma version: 4.1.0
karma-junit-reporter: 1.2.0

### Full stack trace
```
ERROR [karma-server]: Error: Invalid character (�) in string: Expected '😀' to be '1'.
    at UserContext.&lt;anonymous&gt; (src/app/category-admin/services/category-admin.service.spec.ts:234:20)
    at ZoneDelegate../node_modules/zone.js/dist/zone.js.ZoneDelegate.invoke (node_modules/zone.js/dist/zone.js:391:1)
    at ProxyZoneSpec.push../node_modules/zone.js/dist/zone-testing.js.ProxyZoneSpec.onInvoke (node_modules/zone.js/dist/zone-testing.js:289:1)
    at ZoneDelegate../node_modules/zone.js/dist/zone.js.ZoneDelegate.invoke (node_modules/zone.js/dist/zone.js:390:1)
 at index 10
    at XMLStringifier.module.exports.XMLStringifier.assertLegalChar (/Users/sergeyf/job/case-management-frontend/node_modules/karma-junit-reporter/node_modules/xmlbuilder/lib/XMLStringifier.js:153:15)
    at XMLStringifier.assertLegalChar (/Users/sergeyf/job/case-management-frontend/node_modules/karma-junit-reporter/node_modules/xmlbuilder/lib/XMLStringifier.js:4:59)
    at XMLStringifier.module.exports.XMLStringifier.eleText (/Users/sergeyf/job/case-management-frontend/node_modules/karma-junit-reporter/node_modules/xmlbuilder/lib/XMLStringifier.js:33:19)
    at new XMLText (/Users/sergeyf/job/case-management-frontend/node_modules/karma-junit-reporter/node_modules/xmlbuilder/lib/XMLText.js:17:35)
    at XMLElement.module.exports.XMLNode.text (/Users/sergeyf/job/case-management-frontend/node_modules/karma-junit-reporter/node_modules/xmlbuilder/lib/XMLNode.js:165:15)
    at XMLElement.module.exports.XMLNode.node (/Users/sergeyf/job/case-management-frontend/node_modules/karma-junit-reporter/node_modules/xmlbuilder/lib/XMLNode.js:157:15)
    at XMLElement.module.exports.XMLNode.element (/Users/sergeyf/job/case-management-frontend/node_modules/karma-junit-reporter/node_modules/xmlbuilder/lib/XMLNode.js:102:28)
    at XMLElement.module.exports.XMLNode.ele (/Users/sergeyf/job/case-management-frontend/node_modules/karma-junit-reporter/node_modules/xmlbuilder/lib/XMLNode.js:357:19)
    at /Users/sergeyf/job/case-management-frontend/node_modules/karma-junit-reporter/index.js:165:14
    at Array.forEach (<anonymous>)
    at JUnitReporter.specSuccess.specSkipped.specFailure (/Users/sergeyf/job/case-management-frontend/node_modules/karma-junit-reporter/index.js:164:18)
    at JUnitReporter.BaseReporter.onSpecComplete (/Users/sergeyf/job/case-management-frontend/node_modules/karma/lib/reporters/base.js:109:12)
    at Server.<anonymous> (/Users/sergeyf/job/case-management-frontend/node_modules/karma/lib/events.js:40:26)
    at emitTwo (events.js:131:20)
    at Server.emit (events.js:214:7)
    at Browser.onResult (/Users/sergeyf/job/case-management-frontend/node_modules/karma/lib/browser.js:164:20)
    at Socket.socket.on (/Users/sergeyf/job/case-management-frontend/node_modules/karma/lib/browser.js:214:42)
    at emitOne (events.js:116:13)
    at Socket.emit (events.js:211:7)
    at /Users/sergeyf/job/case-management-frontend/node_modules/socket.io/lib/socket.js:528:12
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

Issue moved from https://github.com/karma-runner/karma/issues/3323

**Comment by hicom150** [comment]
This issue has been already fixed in https://github.com/karma-runner/karma-junit-reporter/pull/170 but hasn't been released yet 😅

I think it would be nice if we could **release a new version** including this fix.

@johnjbarton could you guide me in the process of creating a new release? 😉

**Comment by johnjbarton** [comment]
Unfortunately the (first) release is rather involved because of multiple permissions needed. I can look in to helping you get set up in a week.

In the meantime I tried to install and release but `npm install` fails with
```sh
build error 
gyp ERR! stack Error: `make` failed with exit code: 69
gyp ERR! stack     at ChildProcess.onExit (/Users/johnjbarton/.nvm/versions/node/v10.8.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:238:12)
gyp ERR! System Darwin 18.6.0
gyp ERR! command "/Users/johnjbarton/.nvm/versions/node/v10.8.0/bin/node" "/Users/johnjbarton/.nvm/versions/node/v10.8.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /work/karma-junit-reporter/node_modules/libxmljs-mt
gyp ERR! node -v v10.8.0
gyp ERR! node-gyp -v v3.7.0
gyp ERR! not ok 
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! libxmljs-mt@0.16.1 install: `node-gyp rebuild`
npm ERR! Exit status 1
```
Do you know anything about this issue?

**Comment by hicom150** [comment]
This issue should be fixed with #166 so once merged I think we are ready to start preparing the release 💪
